bob-common/Cargo.toml

```diff
@@ -23,6 +24,7 @@ metrics = { version = "0.17", features = ["std"] }
 metrics-exporter-prometheus = { version = "0.6", features = ["tokio-exporter"]}
 metrics-util = "0.10"
 mockall = "0.10"
+parking_lot = "0.12.0"
```

Comment:
In recent versions of Rust, the standard Mutex has become much faster. It is even slightly faster than in the parking_lot one. So we decided to use standard implementation everywhere and you should drop this dependency